django/core/management/commands/makemigrations.py

```diff
@@ -345,7 +346,10 @@ def write_migration_files(self, changes, update_previous_migration_paths=None):
                     # Display a relative path if it's below the current working
                     # directory, or an absolute path otherwise.
                     migration_string = self.get_relative_path(writer.path)
-                    self.log("  %s\n" % self.style.MIGRATE_LABEL(migration_string))
+                    if self.check_changes:
+                        self.log("  %s\n" % app_label)
+                    else:
+                        self.log("  %s\n" % self.style.MIGRATE_LABEL(migration_string))
                     for operation in migration.operations:
                         self.log("    - %s" % operation.describe())
                     if self.scriptable:
```

Comment:
I'm assuming that the line below this (gh won't let me comment on it!) should stay as is with --check